ui/pages/home/home.component.js

```diff
@@ -345,8 +346,49 @@ export default class Home extends PureComponent {
       setWaitForConfirmDeepLinkDialog(false);
     });
     ///: END:ONLY_INCLUDE_IF
+
+    this.triggerNewPrivacyPolicyToast();
   }
 
+  triggerNewPrivacyPolicyToast() {
+    const newPrivacyPolicyDate = new Date(2024, 5, 6);
+
+    const userHasClosedToast = Boolean(
+      window.localStorage?.getItem('privacyPolicyToastClosed'),
+    );
+
+    const shownDate = window.localStorage?.getItem(
+      'privacyPolicyToastShownDate',
+    );
+    const parsedShownDate = shownDate && new Date(Number(shownDate));
+    const currentDate = new Date();
+
+    const oneDayInMilliseconds = 24 * 60 * 60 * 1000;
+
+    if (userHasClosedToast) {
+      this.setState({ showPrivacyPolicyToast: false });
```

Comment:
When `showPrivacyPolicyToast` gets moved to global state, we'll no longer need this local state.